tool/create_code_with_tooltips.dart

```diff
@@ -188,19 +190,24 @@ Iterable<List<String>> _tooltips(String toolTipDataPath) sync* {
     _log.info('  ${match == null ? "Skipping" : "Processing"}: $line.');
     if (match == null) continue;
 
-    String name = match[1],
-        optionalTitle = match[3] ?? '',
-        tooltip = match[4],
-        liClosingTag = match[5];
+    String name = match[1] ?? '';
+    String optionalTitle = match[3] ?? '';
+    String tooltip = match[4] ?? '';
+    String? liClosingTag = match[5];
     _log.fine('  >> $name | $optionalTitle | $tooltip | $liClosingTag');
 
     while (liClosingTag == null && lines.moveNext()) {
       final line = lines.current;
       final match = _tooltipLineEndRE.firstMatch(line);
       _log.info(
           '  ${match == null ? "Skipping data:" : "Processing data:"}: $line.');
-      tooltip = _join(tooltip, match[1]);
-      liClosingTag = match[2];
+      if (match == null) {
+        _log.info('Skipping data: $line');
```

Comment:
We're now duplicating the logged information. I'm guessing you intended to remove lines 202-203.

Also, although it was probably a bug that the previous logged info contained two colons, I'd want confirmation of that just in case it's important for some reason.

Reply:
Yeah it was a bug as far as I can tell, we don't use the output for anything besides logging purposes.